tools/windows.py

```diff
@@ -11,6 +11,7 @@ def options(opts):
     opts.Add(BoolVariable("use_mingw", "Use the MinGW compiler instead of MSVC - only effective on Windows", False))
     opts.Add(BoolVariable("use_clang_cl", "Use the clang driver instead of MSVC - only effective on Windows", False))
     opts.Add(BoolVariable("use_static_cpp", "Link MinGW/MSVC C++ runtime libraries statically", False))
+    opts.Add(BoolVariable("use_asan", "Use address sanitizer (ASAN)", False))
```

Comment:
Why no tsan and others for windows?

Reply:
MSVC doesn't support any other sanitizer.